data/json/snippets/art.json

```
@@ -0,0 +1,205 @@
+[
+  {
+    "type": "snippet",
+    "category": "<sketch_concept_description>",
+    "text": [
+      { "id": "sketch_concept_description_0001", "text": "<physical_build_adjective> <character_gender_adjective>" },
+      { "id": "sketch_concept_description_0002", "text": "<physical_build_adjective> <character_gender_adjective>" },
+      {
+        "id": "sketch_concept_description_0003",
+        "text": "<physical_build_adjective>, <physical_appearance_adjective> <character_gender_adjective>"
+      },
+      {
+        "id": "sketch_concept_description_0004",
+        "text": "<physical_appearance_adjective> <character_gender_adjective>"
+      },
+      {
+        "id": "sketch_concept_description_0005",
+        "text": "<physical_appearance_adjective> anthropomorphic <nonhuman_species_noun> <character_gender_adjective>"
+      },
+      { "id": "sketch_concept_description_0006", "text": "<nonhuman_species_noun>" },
+      {
+        "id": "sketch_concept_description_0007",
+        "text": "<physical_appearance_adjective> <humanoid_species_noun> <character_gender_adjective>"
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<physical_build_adjective>",
+    "//": "More objective adjectives of someone's physical build.",
+    "text": [
+      { "id": "physical_build_adjective_0001", "text": "buff" },
```

Short phrases should be specified with translation context to avoid conflicting with identical words with different meanings when translating. For example:

```suggestion
      { "id": "physical_build_adjective_0001", "text": { "str": "buff", "ctxt": "physical build" } },
```